processing/src/main/java/io/druid/query/groupby/GroupByQuery.java

```diff
@@ -308,6 +308,7 @@ public GroupByQuery withDimensionSpecs(final List<DimensionSpec> dimensionSpecs)
     private LimitSpec limitSpec = null;
     private List<OrderByColumnSpec> orderByColumnSpecs = Lists.newArrayList();
     private int limit = Integer.MAX_VALUE;
+    private QueryGranularity orderingGranularity;
```

Comment:
what happens if user specified orderingGranularity be smaller than the granularity ? do we need to put a check?